cvat/apps/engine/tests/test_rq.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import unittest
+import uuid
+from unittest import mock
+
+import django_rq
+from django.conf import settings
+from django_rq.queues import DjangoRQ
+from rq.job import Dependency, Job
+
+from cvat.apps.engine.rq import define_dependent_job
+from cvat.apps.engine.tests.utils import clear_rq_jobs
+
+DEFAULT_USER_ID = 1
+
+
+def dummy_task():
+    return 2 + 2
+
+
+def _enqueue_test_job(
+    queue: DjangoRQ,
+    job_id: str | None = None,
+    user_id: int = DEFAULT_USER_ID,
+    depends_on: list[Job] | None = None,
+) -> Job:
+    job_id = job_id or str(uuid.uuid4())
```

Comment:
Is there any reason to define such a custom job_id? (here and in other places)

Reply:
No, I don't think so, all tests can be updated to use ids of created jobs

FYI, we do not merge PRs without approvals